cayleypy/datasets.py

```diff
@@ -108,4 +112,5 @@ def generate_datasets():
     _update_dataset("all_transpositions_cayley_growth", keys, _compute_all_transpositions_cayley_growth)
     keys = [str(n) for n in range(2, 11)]
     _update_dataset("pancake_cayley_growth", keys, _compute_pancake_cayley_growth)
+    _update_dataset("burnt_pancake_cayley_growth", keys, _compute_burnt_pancake_cayley_growth)
```

Comment:
Add a line `keys = [str(n) for n in range(1, 11)]` above, because case n=1 is valid for this problem, and expected growth function for it is [1,1].